### ISSUE TITLE
ProgressIndicator : aria-valuetext isn't read by Narrator in scan mode

### ISSUE THREAD

**Issue by dPJinjo** [OfficeDev/office-ui-fabric-react#11079]
### Environment Information

- **Package version(s)**: 7.56.0
- **Browser and OS versions**: Edge Version 79.0.309.11; Windows 18363.418; Narrator 

### Describe the issue:

ProgressIndicator with ariaValueText attribute set doesn't narrate the string under Narrator. I couldn't get this to work with any version of Edge I have on my machine.

### Please provide a reproduction of the issue in a codepen:

* Why doesn't codepen render fabric controls right now?
* Can repro in control documentation page using browser developer tools forcibly add the aria-valuetext attribute and give it a value. It also doesn't work in actual code, but I can't get codepen to render fabric controls and I don't know why. Even old codepens that used to work seem to render nothing anymore.

#### Actual behavior:

Since ProgressIndicator isn't a tab-stop, I put Narrator in scan-mode and keyboard navigated to the control using the keyboard arrow keys. When the progress bar has focus, Narrator reads "progress bar 70%". I set ariaValueText="robert", but never heard this read. I see in the browser developer tools that the aria-valuetext property is set to "robert", but I never hear it.

#### Expected behavior:

At some point, I would expect to hear the ariaValueText value read. Interestingly aria-labelledby does work if I use the developer tools to assign the attribute value, but aria-valuetext doesn't seem to do anything.

### Documentation describing expected behavior

https://developer.microsoft.com/en-us/fabric#/controls/web/progressindicator#implementation

**Comment by JasonGore** [comment]
Actually this appears to be a bit different from #7993. I'll take a closer look.

**Comment by JasonGore** [comment]
I set up a [basic CodePen](https://codepen.io/jasongore/pen/dyymNYX) reproducing this issue with a simple `progressbar` without any Fabric controls. Narrator seems to violate [ARIA standards](https://www.w3.org/TR/wai-aria-1.1/#aria-valuetext) that say `aria-valuetext` should be dictated over `aria-valuenow` when provided. I've entered an issue in Narrator repo as well. I'll resolve this as external for now.
